modules/ROOT/pages/intro-mule-message.adoc

```diff
@@ -94,14 +94,18 @@ default value for a variable if the actual value resolves to `null`.
 
 == Session Properties
 Session properties are no longer needed and have been removed in Mule 4.
-You should store data in Mule event variables.
+Instead, store data in target variables (sometimes called target parameters). 
 
 These properties are not needed because Mule 3 transports and their associated
 transport barriers do not exist in Mule 4. Mule 3 transport barriers determine
 how to handle Mule message data that moves through Mule 3 flows. Transports are
 replaced by corresponding Mule connectors (such as the FTP connector) in Mule 4,
 which handle networking transport protocols differently.
 
+For examples, see 
+xref:migration-core-enricher.adoc[Migrating the Enricher to a Target Parameters]
```

Comment:
a target parameter or target parameters? Line 106

Reply:
The terminology is not consistent across the docs. We'll handle in a separate Jira.